src/index.ts

```diff
@@ -73,6 +81,10 @@ export function autoRetry(
                 typeof result.parameters?.retry_after === 'number' &&
                 result.parameters.retry_after <= maxDelay
             ) {
+                if (logging === true) {
+                    const timeString = new Date().toLocaleString();
+                    console.log(`[${timeString}] Retrying after ${result.parameters.retry_after} seconds`);
```

Comment:
We should include in the log message the name of the method which is being called again